src/appState.ts

```diff
@@ -38,6 +38,7 @@ export function getDefaultAppState(): AppState {
     selectedElementIds: {},
     collaborators: new Map(),
     shouldCacheIgnoreZoom: false,
+    fullScreenEnabled: false,
```

Comment:
Can you use the browser state instead of reinventing your own. 
https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement

Right now if you open full screen and then press escape, the full screen button is still shown as fullscreen rather than normal.